tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -405,11 +405,10 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
       |  }
       |}
     """.stripMargin,
-    """|head :: next scala.collection.immutable
-       |Nil scala.collection.immutable""".stripMargin,
+    "",
     compat = Map(
-      "2.12" ->
-        """|head :: tl scala.collection.immutable
+      "3" ->
+        """|head :: next scala.collection.immutable
```

Comment:
Shouldn't Scala 3 also return empty?

Reply:
It should, but this PR solves some other issue. Previously in `case abc@@` we weren't doing fuzzy match, because `abc` was automatically treated as name for bind `abc @ something`, now we use `abc` as pattern if there is no `@`. With `new` and scala 3 the issue remains, because there is no infomation about `new` in the tree, it's neither a bind or ident